requirements.in

```diff
@@ -1,7 +1,8 @@
-boto3
 tqdm
 arrow
 pandas
 daiquiri
 google-cloud-bigquery
 lxml
+s3fs
+-e git+git://github.com/SohierDane/BigQuery_Helper#egg=bq_helper
```

Comment:
@arajkumar I can see this last commit to this repository was in 2018, the big query API has changed quite a bit in that time. Are you sure you guys want to use it?